recipes/bowtie/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "bowtie" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "a9edd8081257ab96bb820379b501ab26f310c489d0d716830b8c08fd9aa392b6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
```

Comment:
There are a bunch of things [in `requirements.txt`](https://github.com/jwkvam/bowtie/blob/master/requirements.txt) that aren't here. Is that a problem?

Reply:
[Apparently yes](https://circleci.com/gh/conda-forge/staged-recipes/23778?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), given the `ModuleNotFoundError: No module named 'astor'` error.

Reply:
yea those are really for testing, actual requirements are in `pyproject.toml`. I recently removed astor but forgot 0.7.0 still requires it.

Reply:
@dougalsutherland Thanks, I think this is good to go now.